scripts/jobs/validate/partest

```diff
@@ -4,5 +4,5 @@ baseDir=${WORKSPACE-`pwd`}
 scriptsDir="$baseDir/scripts"
 . $scriptsDir/common
 
-sbt $sbtArgs update compile "partest-only run"
+sbt $sbtArgs update compile "partest-only run --show-diff --verbose"
```

Comment:
@smarter the diffs always get shown at the end because of --verbose, what wasn't working was the inline diffs (right where the test fails). It's possible that it works on your machine, partest tries to use git and some other tools to get the best diff it can, so depending on your environment you might get different results.

It seems that the one used for me and some other people doesn't get appended to the test transcript the way the reporting expects, that's why it didn't get printed. The fix is here: https://github.com/lampepfl/dotty/pull/612